README.md

```diff
@@ -1,24 +1,26 @@
-# Frobnicator (this is a template)
+[![Review Assignment Due Date](https://classroom.github.com/assets/deadline-readme-button-22041afd0340ce965d47ae6ef1cefeee28c7c493a6346c4f15d667ab976d596c.svg)](https://classroom.github.com/a/M0kyOMLZ)
+# JuweAppka
 
-## Authors
-- Andrzej Głuszak (@agluszak on GitHub)
-- Linus Torvalds (@torvalds on GitHub)
+## Autorzy
+- Błażej Wilkoławski (@Buarzej on GitHub)
+- Paulina Kubera (@c00kiepreferences on GitHub)
 
-## Description
-Frobnicator is going to be a platformer game similar to Super Mario Bros made using Korge game engine.
+## Opis
+JuweAppka to aplikacja mobilna przeznaczona na systemy Android i iOS będąca aplikacją festiwalu muzycznego "Juwenalia UW".
 
-## Features
-- map generator
-- shooting
-- enemy AI
-- game state saving and loading
-- scores
+## Funkcje
+- harmonogram koncertów i innych wydarzeń towarzyszących w dniach festiwalu
+- lista aktualnie ogłoszonych artystów oraz możliwość odsłuchania ich najpopularniejszych utworów bezpośrednio w aplikacji
```

Comment:
w jaki konkretnie sposób chcecie zaimplementować możliwość odsłuchania utworów? zaembeddowana strona z youtube'a/spotify? czy chcecie napisać odtwarzacz muzyki wewnątrz aplikacji ze ściąganiem mp3 z internetu?

Reply:
Wstępny plan to zaembeddowany odtwarzacz YouTube (np. przy użyciu [tej biblioteki](https://github.com/PierfrancescoSoffritti/android-youtube-player)), ale jeśli nie starczy na to czasu, to zaimplementujemy zwykłą funkcjonalność odtwarzacza MP3 z wgranymi plikami audio.